### ISSUE TITLE
Performance of baseline

### ISSUE THREAD

**Issue by Luffy03** [eccv22-ood-workshop/ROBIN-dataset#3]
Hello, thanks for your great work first!
I run your provided baseline directly but get low performance. The results are quite different with that reported in your paper. Is there sth wrong with the code?
![屏幕截图 2022-06-10 171609](https://user-images.githubusercontent.com/52514670/173033976-e0bc1bf9-c032-4c2d-baad-06ed9764b73e.png)
![屏幕截图 2022-06-10 171720](https://user-images.githubusercontent.com/52514670/173033995-4b36501f-1a32-4f68-a004-fb430c52837e.png)

**Comment by DTennant** [comment]
Hi, the released dataset is only the validation set, not the full test set we used for the paper.
We will open a codalab competition later which will enable people to test their model on the full test set.

Also the code in this repo should be able to reach a 60% or so performance on the validation set that I released using this command:

```
python3 main.py --data /path/to/download/ROBIN-cls-train/ --val-data /path/to/download/ROBIN-cls-val/ --pretrained
```

**Comment by Luffy03** [comment]
> Hi, the released dataset is only the validation set, not the full test set we used for the paper. We will open a codalab competition later which will enable people to test their model on the full test set.
> 
> Also the code in this repo should be able to reach a 60% or so performance on the validation set that I released using this command:
> 
> ```
> python3 main.py --data /path/to/download/ROBIN-cls-train/ --val-data /path/to/download/ROBIN-cls-val/ --pretrained
> ```

I do follow this construction but the top-1 acc on val set is very low, as shown in the fig.......

**Comment by Luffy03** [comment]
The acc on the training set is normal
![屏幕截图 2022-06-10 220139](https://user-images.githubusercontent.com/52514670/173082691-dedf54db-4ddd-48e3-a7fa-4b9e6cbb99c5.png)

**Comment by Luffy03** [comment]
Would you please present the detailed results of your baseline method on the val set? It will help me a lot! thx!

**Comment by DTennant** [comment]
Hi, we will update the repo with more information ASAP.
For now, we are still preparing the data, please stay tuned.

**Comment by Luffy03** [comment]
> Hi, we will update the repo with more information ASAP. For now, we are still preparing the data, please stay tuned.

Thank you very much! Looking forward to it.

**Comment by DTennant** [comment]
Hi,
Thanks again for your interest in our workshop, and we apologise for the late update.
I have update the README in this repo and update a link to the full test set and labels in the README, the codalab submission server will also be online within this week.

Here are some changes to the competition:

The Phase-1 of the competition will not be a code submission challenge, we have released all the test data and labels in this repo. And Phase-1 will last longer than original planed, we will ask each team to provide a description of their developing environment at the end of Phase-1, Phase-2 will still be code submission challenge.
We will be using Top-1 accuracy for image-classification, mAP@50 for object detection, and Acc@pi/6 for pose estimation as the metric, the IID test performance will also be considered as per request of the sponsor, we will penalize submissions that are significantly different in IID performance with our baseline.
The only limitation now is that the model should only be trained on the given training set and/or the ImageNet-1k dataset, no additional dataset is allowed. You can use any ensemble, data augmentation, or test-time training techniques.
Thanks again for your patience.

Best regards

**Comment by khawar-islam** [comment]
Hello @DTennant 

Could you please share the "full test set you used for the paper." I have compare your results with my results and thus i need full test dataset fro three tasks img cls, obj detec and pose.
